plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/ApplicationWorkbenchAdvisor.java

```diff
@@ -411,23 +412,23 @@ private boolean saveAndCleanup() {
                 }
             }
 
-            return cancelRunningTasks() && closeActiveTransactions();
+            return ApplicationWorkbenchAdvisor.cancelRunningTasks(true) && ApplicationWorkbenchAdvisor.closeActiveTransactions(false);
```

Comment:
These don't need to be static or public. The class from which you are calling them actually inherits from this particular class; hence, methods can be made `protected`.